packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -213,11 +331,35 @@ const description = computed(() => highlightSearchTerms(model.value?.description
 </script>
 
 <style scoped>
-.graph-element {
-	flex: 1;
-	height: 400px;
+.model-panel {
+	height: 100%;
+}
+.resizable {
+	position: relative;
+	height: 100%;
 	width: 100%;
-	border: 1px solid var(--surface-border);
+	min-height: 200px;
+	border: 1px solid #ccc;
+	overflow: hidden;
+}
+
+.resizer {
+	width: 100%;
+	height: 5px;
+	cursor: ns-resize;
+	background-color: #1c857764;
+	border-radius: 0.25rem;
+}
+
+.content {
+	height: 99%;
+	width: 100%;
+}
+.graph-element {
+	background-color: #f9fafb;
```

Comment:
Please use the CSS properties within `src/assets/theme/_variables.scss`, they are all at the bottom of the file. We do not do hard-coded colours etc.